framework/py/flwr/common/inflatable_test.py

```diff
@@ -14,9 +14,13 @@
 # ==============================================================================
 """Inflatable test."""
 
+from __future__ import annotations
```

Comment:
We usually have two space lines between the module docstring and the first import line.

```suggestion

from __future__ import annotations
```